### ISSUE TITLE
S-164_1.1 V3 : Sounding value is less than the DRVAL1 of the covering bathymetric area

### ISSUE THREAD

**Issue by plebihan29n** [iho-ohi/S-101-Test-Datasets#81]
There are some soundings for which some values are less than the DRVAL1 of their covering areas.
These cases lead to raise  errors when checking cells (S-101_A_145 or 146 for S-158 and 1768 for S-58)
Here is an example for 10100AA_X0000 (or GB4X0000)

![plot-start-expsou](https://github.com/iho-ohi/S-101-Test-Datasets/assets/24851994/851678cb-2732-4c36-9ac0-d86dfdf7469b)

My proposal is to create consistent depth areas around these soundings.
![plot-end-expsou](https://github.com/iho-ohi/S-101-Test-Datasets/assets/24851994/3a8968a0-2462-4c85-a8a9-1acfae6b9e26)

Do you agree with this proposal and is it worth doing now ?

**Comment by plebihan29n** [comment]
An additional remark : This issue could be joined to a more general issue "Correct S-58 errors on the original S-64 test datasets" . The objective is to clean as much as possible the  S-64 test datasets before conversion.

**Comment by kusala9** [comment]
worth checking if any of these aren't required for any tests in S-64. I don't think they are - were they originally marked as EXPSOU=2? I would support getting the datasets compliant with S-58 before conversion as long as it doesn't mess up any of the test conditions (there shouldn't be many, for example the invalid object tests etc...).

**Comment by plebihan29n** [comment]
I don't think also that these errors/warnings are linked to a specific test. Above all, for the moment I try to detect them only on datasets from power-up, which are considered to the equivalent of a valid datasets (it is my understanding).
What I can do for the moment , is just labelled these kind of issues which a label :  "S-58 Errors/Warnings" something like that.
And in a second step, we will decide what to do with "S-58 Errors/Warnings" issues.

**Comment by MikusRL** [comment]
Soundings with shoaler depths EXPSOU=2 are not disallowed, but "strongly advised" not to use by UOC. To my experience these are present in ENCs still quite a few for quite a few countries.
![image](https://github.com/iho-ohi/S-101-Test-Datasets/assets/72803214/54c2e9d4-bb25-4208-907d-49490d4cfcf3)

**Comment by TomRichardson6** [comment]
I agree with others that this stems wider issues with these datasets my suggestion would be to make this an Obstruction or change the depth value to be within the range. I support adding this to a more general issue about validating and improving the datasets. We have seen other cases of odd or very dated encoding in them.

**Comment by kusala9** [comment]
agree with Tom, these need to be looked at. A word of caution though, as some of the S-64 datasets contain combinations that exercise branches of the CSPs and may be there for a reason. Even if they are outlawed by the UOC, they still represent valid combinations and as such require testing as the CSPs determine the ECDIS behaviour so we need to be careful about which ones we delete/modify and also how we reproduce the "equivalent" ones in S-164 (something we're looking at at the moment).

**Comment by plebihan29n** [comment]
I think it could be useful  to keep in mind that S-57 or S-101 datasets from Power Up  	GB4X0000,GB5X01NE,GB5X01SE,GB5X01NW,GB5X01SW,GB5X01SE are datasets  that contain as much as possible exhausting combination of feature type and attributes that respect the encoding guide rules and reflect the reality.
This power up could be useful for valid plots, real scenarios , ...

For other specific and plausible cases (example Soundings with expsou 2), useful for testing behavior of ECDIS we can find another place (DataSet) to encode it.
